WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -256,6 +308,64 @@ public function check_for_matches( $stackPtr ) {
 	} // End check_for_matches().
 
 	/**
+	 * Verify if the current token is one of the targetted functions as callback.
+	 *
+	 * @since 0.11.0
+	 *
+	 * @param int $stackPtr The position of the current token in the stack.
+	 *
+	 * @return int|false
```

Comment:
Please document the return value: `stackPtr if for where to skip to if a callback was found, false if the function call is not a callback.`